src/deploy/functions/args.ts

```diff
@@ -1,13 +1,15 @@
 import * as backend from "./backend";
 import * as gcfV2 from "../../gcp/cloudfunctionsv2";
 import * as projectConfig from "../../functions/projectConfig";
+import * as deployHelper from "./functionsDeployHelper";
 
 // These types should proably be in a root deploy.ts, but we can only boil the ocean one bit at a time.
 
 // Payload holds the output types of what we're building.
```

Comment:
If we're adding what we already have, we should probably update this description a bit